deepmd/descriptor/se_a.py

```diff
@@ -735,6 +749,8 @@ def _filter(
             name='linear', 
             reuse=None,
             trainable = True):
+        if self.mixed_prec is not None:
+            inputs = tf.cast(inputs, get_precision(self.mixed_prec['compute_prec']))
```

Comment:
Do we need this line? the `inputs` are anyway cast to `compute_prec` in networks.one_layer or networks.embedding_net

Reply:
1. There's matrix multiplication outside the embedding net, we need to cast the inputs to match the dtype of the embedding net output.
2. Half precision slicing will be more efficient.